File: CPP/program/AnggotaDPR.cpp
// Import Library
#include <iostream>
#include <string>

// using standard namespace.
using namespace std;

class AnggotaDPR
{
private:
    string id;
    string name;
    string bidang;
    string partai;

public:
    AnggotaDPR()
    {
        this->id = "";
        this->name = "";
        this->bidang = "";
        this->partai = "";
    }

    AnggotaDPR(string id, string name, string bidang, string partai)
    {
        this->id = id;
        this->name = name;
        this->bidang = bidang;
        this->partai = partai;
    }

    string get_id()
    {
        return id;
    }

    void set_id(string id)
    {
        this->id = id;
    }

    string get_name()
    {
        return this->name;
    }

    void set_name(string name)
    {
        this->name = name;
    }

    string get_bidang()
    {
        return this->bidang;
    }

    void set_bidang(string bidang)
    {
        this->bidang = bidang;
    }

    string get_partai()
    {
        return this->partai;
    }

    void set_partai(string partai)
    {
        this->partai = partai;
    }

    void eat()
    {
        cout << this->name << " is eating!" << '\n';
    }

    void sleep()
    {
        cout << this->name << " is sleeping!" << '\n';
    }

    ~AnggotaDPR() {}
};

File: PHP/AnggotaDPR.php
<?php
class AnggotaDPR {
    // Properti untuk menyimpan data anggota DPR
    private $id = "";
    private $name = "";
    private $bidang = "";
    private $partai = "";

    // Konstruktor untuk membuat objek AnggotaDPR
    public function __construct($id = "", $name = "", $bidang = "", $partai = "") {
        $this->id = $id;
        $this->name = $name;
        $this->bidang = $bidang;
        $this->partai = $partai;
    }

    // Getter & Setter methods untuk mengakses dan mengubah nilai properti
    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getBidang() {
        return $this->bidang;
    }

    public function setBidang($bidang) {
        $this->bidang = $bidang;
    }

    public function getPartai() {
        return $this->partai;
    }

    public function setPartai($partai) {
        $this->partai = $partai;
    }
}
?>
